pwnlib/rop/srop.py

```diff
@@ -1,4 +1,17 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+
+from collections import namedtuple
+
+from pwnlib.abi import ABI
+from pwnlib.context import LocalContext
+from pwnlib.context import context
+from pwnlib.log import getLogger
+from pwnlib.util.packing import flat
+from pwnlib.util.packing import pack
+from pwnlib.util.packing import unpack_many
+
```

Comment:
Is the docstring still present in `rop.srop.__doc__`?

Reply:
That is weird that by moving the imports to the top of the file stopped some of the tests from running. I am reverting it back to the original.

Reply:
As you can see in the [official definition](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring), a string gets into `__doc__` only if it is the first statement of a code (here it is after import statements).